contrib/scripts/kind.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+# Sample usages
+# ./contrib/scripts/kind.sh
+# IPv6=1 ./contrib/scripts/kind.sh
+# NO_BUILD=1 ./contrib/scripts/kind.sh
+# NO_PROVISION=1 ./contrib/scripts/kind.sh
```

Comment:
You could check for the number of arguments passed to the script, and print a help message shortly describing those variables if the user tries to pass any argument (which would likely be `-h` or `--help` if they haven't opened the script to read it).

Reply:
Good point :100: